src/components/ha-selector/ha-selector-select.ts

```diff
@@ -194,6 +195,16 @@ export class HaSelectSelector extends LitElement {
           `
         )}
       </ha-select>
+      ${this.value
+        ? html`<ha-svg-icon
```

Comment:
You should import `ha-svg-icon`

Reply:
And it should be an `ha-icon-button`

Reply:
Check the password button: https://github.com/home-assistant/frontend/blob/dev/src/components/ha-selector/ha-selector-text.ts#L76